index.html

```diff
@@ -267,7 +267,63 @@ <h3>LET'S CONNECT</h3>
                     </div>
                 </div>
             </section>
+
         </main>
+        <footer>
+            <section class = "contact">
+                <div class="contact-1">
+                    <h1 id = "contact-me">
+                        Contact me
+                    </h1>
+                    <p id ="contact-messeage">
+                        If you have an application you <br>
+                         are interested in developing, a <br>
+                         feature that you need built or a <br>
+                          project that needs
+                        coding. I’d <br>
+                         love to help with it.
+                    </p>
+                    <form action="https://formspree.io/f/%7Bform_id%7D"
+                      method="post"
+                      class="my-contact-form">
+                        <ul id = "form-data">
+                            <li class ="input-data">
+                               <input
+                                 type="text"
+                                 id="name"
```

Comment:
- [ ] Well done in implementing the changes said by the previous reviewer. Everything is good the only change you need to make is padding for a placeholder.  So kindly implement it as this is your last chance to submit the project So I am giving it optional.
### your design
![your](https://user-images.githubusercontent.com/81727988/177784259-015d37b0-5b1a-4f3f-bf90-e97550ed63a8.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/177784301-abb031f9-d462-49f1-9405-a47979e0d0b9.png)